### ISSUE TITLE
[ISSUE]: W-Okada doesn't use RX 9060XT 16GB

### ISSUE THREAD

**Issue by Haorelian** [w-okada/voice-changer#1583]
### Voice Changer Version

MMVCServerSIO_win_onnxdirectML-cuda_v.1.5.3.18a

### Operational System

Windows 11 25H2

### GPU

RX 9060XT 16GB

### Read carefully and check the options

- [ ] I've tried to Clear Settings
- [ ] Sample/Default Models are working
- [x] I've tried to change the Chunk Size
- [x] GUI was successfully launched
- [x] I've read the [tutorial](https://github.com/w-okada/voice-changer/blob/master/tutorials/tutorial_rvc_en_latest.md)
- [x] I've tried to extract to another folder (or re-extract) the .zip file

### Model Type

RVC

### Issue Description

In UI no matter which GPU (GPU 0-3) I select it doesn't use my GPU unfortunately and hammers my CPU down to hell around %90 usage. Maybe unsupported GPU?

Also tried V2 but it doesn't even launch.

### Application Screenshot

<img width="764" height="1011" alt="Image" src="https://github.com/user-attachments/assets/3e181236-968e-448a-a602-930b1e3ec353" />

### Logs on console

C:\MMVCServerSIO>MMVCServerSIO.exe     -p 18888     --https false     --content_vec_500 pretrain/checkpoint_best_legacy_500.pt     --content_vec_500_onnx pretrain/content_vec_500.onnx     --content_vec_500_onnx_on true     --hubert_base pretrain/hubert_base.pt     --hubert_base_jp pretrain/rinna_hubert_base_jp.pt     --hubert_soft pretrain/hubert/hubert-soft-0d54a1f4.pt     --nsf_hifigan pretrain/nsf_hifigan/model     --crepe_onnx_full pretrain/crepe_onnx_full.onnx     --crepe_onnx_tiny pretrain/crepe_onnx_tiny.onnx     --rmvpe pretrain/rmvpe.pt     --model_dir model_dir     --samples samples.json
    Booting PHASE :__main__
    PYTHON:3.10.11 (tags/v3.10.11:7d4cc5a, Apr  5 2023, 00:38:17) [MSC v.1929 64 bit (AMD64)]
    Activating the Voice Changer.
                                                  [Voice Changer] download sample catalog. samples_0004_t.json
[Voice Changer] download sample catalog. samples_0004_o.json
[Voice Changer] download sample catalog. samples_0004_d.json
SlotInfo::: RVCModelSlot(slotIndex=-1, voiceChangerType='RVC', name='つくよみちゃん(onnx)', description='', credit='つくよみちゃん', termsOfUseUrl='https://huggingface.co/wok000/vcclient_model/raw/main/rvc_v2_alpha/tsukuyomi-chan/terms_of_use.txt', iconFile='tsukuyomi-chan.png', speakers={0: 'target'}, modelFile='tsukuyomi_v2_40k_e100_simple.onnx', indexFile='', defaultTune=0, defaultIndexRatio=0, defaultProtect=0.5, isONNX=True, modelType='pyTorchRVC', samplingRate=-1, f0=True, embChannels=256, embOutputLayer=9, useFinalProj=True, deprecated=False, embedder='hubert_base', sampleId='Tsukuyomi-chan_o', version='v2')
SlotInfo::: RVCModelSlot(slotIndex=-1, voiceChangerType='RVC', name='あみたろ(onnx)', description='', credit='あみたろ', termsOfUseUrl='https://huggingface.co/wok000/vcclient_model/raw/main/rvc_v2_alpha/amitaro/terms_of_use.txt', iconFile='amitaro.png', speakers={0: 'target'}, modelFile='amitaro_v2_40k_e100_simple.onnx', indexFile='', defaultTune=0, defaultIndexRatio=0, defaultProtect=0.5, isONNX=True, modelType='pyTorchRVC', samplingRate=-1, f0=True, embChannels=256, embOutputLayer=9, useFinalProj=True, deprecated=False, embedder='hubert_base', sampleId='Amitaro_o', version='v2')
SlotInfo::: RVCModelSlot(slotIndex=-1, voiceChangerType='RVC', name='黄琴まひろ(onnx)', description='', credit='黄琴まひろ', termsOfUseUrl='https://huggingface.co/wok000/vcclient_model/raw/main/rvc_v2_alpha/kikoto_mahiro/terms_of_use.txt', iconFile='kikoto_mahiro.png', speakers={0: 'target'}, modelFile='kikoto_mahiro_v2_40k_simple.onnx', indexFile='', defaultTune=0, defaultIndexRatio=0, defaultProtect=0.5, isONNX=True, modelType='pyTorchRVC', samplingRate=-1, f0=True, embChannels=256, embOutputLayer=9, useFinalProj=True, deprecated=False, embedder='hubert_base', sampleId='KikotoMahiro_o', version='v2')
SlotInfo::: RVCModelSlot(slotIndex=-1, voiceChangerType='RVC', name='刻鳴時雨(onnx)', description='', credit='刻鳴時雨', termsOfUseUrl='https://huggingface.co/wok000/vcclient_model/raw/main/rvc_v2_alpha/tokina_shigure/terms_of_use.txt', iconFile='tokina_shigure.png', speakers={0: 'target'}, modelFile='tokina_shigure_v2_40k_e100_simple.onnx', indexFile='', defaultTune=0, defaultIndexRatio=0, defaultProtect=0.5, isONNX=True, modelType='pyTorchRVC', samplingRate=-1, f0=True, embChannels=256, embOutputLayer=9, useFinalProj=True, deprecated=False, embedder='hubert_base', sampleId='TokinaShigure_o', version='v2')
[Voice Changer] Downloading model files...
[Voice Changer] Generating metadata...
[Voice Changer] ONNX Model: Official v2 like
SlotInfo::: RVCModelSlot(slotIndex=0, voiceChangerType='RVC', name='つくよみちゃん(onnx)', description='', credit='つくよみちゃん', termsOfUseUrl='https://huggingface.co/wok000/vcclient_model/raw/main/rvc_v2_alpha/tsukuyomi-chan/terms_of_use.txt', iconFile='tsukuyomi-chan.png', speakers={'0': 'target'}, modelFile='tsukuyomi_v2_40k_e100_simple.onnx', indexFile='', defaultTune=0, defaultIndexRatio=0, defaultProtect=0.5, isONNX=True, modelType='onnxRVC', samplingRate=40000, f0=True, embChannels=768, embOutputLayer=12, useFinalProj=False, deprecated=False, embedder='hubert_base', sampleId='Tsukuyomi-chan_o', version='v2')
[Voice Changer] ONNX Model: Official v2 like
SlotInfo::: RVCModelSlot(slotIndex=1, voiceChangerType='RVC', name='あみたろ(onnx)', description='', credit='あみたろ', termsOfUseUrl='https://huggingface.co/wok000/vcclient_model/raw/main/rvc_v2_alpha/amitaro/terms_of_use.txt', iconFile='amitaro.png', speakers={'0': 'target'}, modelFile='amitaro_v2_40k_e100_simple.onnx', indexFile='', defaultTune=0, defaultIndexRatio=0, defaultProtect=0.5, isONNX=True, modelType='onnxRVC', samplingRate=40000, f0=True, embChannels=768, embOutputLayer=12, useFinalProj=False, deprecated=False, embedder='hubert_base', sampleId='Amitaro_o', version='v2')
[Voice Changer] ONNX Model: Official v2 like
SlotInfo::: RVCModelSlot(slotIndex=2, voiceChangerType='RVC', name='黄琴まひろ(onnx)', description='', credit='黄琴まひろ', termsOfUseUrl='https://huggingface.co/wok000/vcclient_model/raw/main/rvc_v2_alpha/kikoto_mahiro/terms_of_use.txt', iconFile='kikoto_mahiro.png', speakers={'0': 'target'}, modelFile='kikoto_mahiro_v2_40k_simple.onnx', indexFile='', defaultTune=0, defaultIndexRatio=0, defaultProtect=0.5, isONNX=True, modelType='onnxRVC', samplingRate=40000, f0=True, embChannels=768, embOutputLayer=12, useFinalProj=False, deprecated=False, embedder='hubert_base', sampleId='KikotoMahiro_o', version='v2')
[Voice Changer] ONNX Model: Official v2 like
SlotInfo::: RVCModelSlot(slotIndex=3, voiceChangerType='RVC', name='刻鳴時雨(onnx)', description='', credit='刻鳴時雨', termsOfUseUrl='https://huggingface.co/wok000/vcclient_model/raw/main/rvc_v2_alpha/tokina_shigure/terms_of_use.txt', iconFile='tokina_shigure.png', speakers={'0': 'target'}, modelFile='tokina_shigure_v2_40k_e100_simple.onnx', indexFile='', defaultTune=0, defaultIndexRatio=0, defaultProtect=0.5, isONNX=True, modelType='onnxRVC', samplingRate=40000, f0=True, embChannels=768, embOutputLayer=12, useFinalProj=False, deprecated=False, embedder='hubert_base', sampleId='TokinaShigure_o', version='v2')
    Internal_Port:18888
    protocol: HTTP
    -- ---- --
    Please open the following URL in your browser.
    http://<IP>:<PORT>/
    In many cases, it will launch when you access any of the following URLs.
    http://127.0.0.1:18888/

[VCClient] Access http://127.0.0.1:18888/
[VCClient] wait web server...0 http://127.0.0.1:18888/
[VCClient] wait web server... done 200
[2025-11-01 14:47:47] connet sid : qJBoNveHcJQ-aZRvAAAC
[2025-11-01 14:47:47] connet sid : V9_wU3jCKBbID4n8AAAD
[Voice Changer] update configuration: serverReadChunkSize 512
[Voice Changer] update configuration: gpu 0
[Voice Changer] update configuration: modelSlotIndex 1761997690000
[Voice Changer] generate new embedder. (no embedder)
[Voice Changer] Loading index...
[Voice Changer] Index file is not found
[Voice Changer] update configuration: tran 8
[Voice Changer] update configuration: tran 8
[Voice Changer] update configuration: tran 10
[Voice Changer] update configuration: tran 12
[Voice Changer] update configuration: tran 13
[Voice Changer] update configuration: tran 14
[Voice Changer] update configuration: tran 15
[Voice Changer] update configuration: tran 17
[Voice Changer] update configuration: tran 19
[Voice Changer] update configuration: tran 20
[Voice Changer] update configuration: tran 21
[Voice Changer] update configuration: tran 22
[Voice Changer] update configuration: tran 23
[Voice Changer] update configuration: tran 23
[Voice Changer] update configuration: tran 23
[Voice Changer] update configuration: tran 24
[Voice Changer] update configuration: tran 25
[Voice Changer] update configuration: tran 26
[Voice Changer] update configuration: tran 27
[Voice Changer] update configuration: tran 28
[Voice Changer] update configuration: tran 27
[Voice Changer] update configuration: tran 26
[Voice Changer] update configuration: tran 26
[Voice Changer] update configuration: tran 25
[Voice Changer] update configuration: tran 24
[Voice Changer] update configuration: tran 23
[Voice Changer] update configuration: tran 22
[Voice Changer] update configuration: tran 21
[Voice Changer] update configuration: tran 20
[Voice Changer] update configuration: tran 19
[Voice Changer] update configuration: tran 20
[Voice Changer] update configuration: tran 20
[Voice Changer] update configuration: modelSlotIndex 1761997768001
Pipeline has been deleted
[Voice Changer] generate new embedder. (anyway)
[Voice Changer] Loading index...
[Voice Changer] Index file is not found
[Voice Changer] update configuration: tran 26
[Voice Changer] update configuration: tran 12

**Comment by Haorelian** [comment]
I've managed to get the V2.0.78 STD version somehow work. DML download version doesn't work.

Here's the V2.0.78 STD's log.

`

                        vcclient

----------------------------------------------------------
 Application | http://localhost:18000
 Log(rich)   | http://localhost:18000/?app_mode=LogViewer
 Log(text)   | http://localhost:18000/vcclient.log
 API         | http://localhost:18000/docs
 License(js) | http://localhost:18000/licenses-js.json
 License(py) | http://localhost:18000/licenses-py.json
----------------------------------------------------------
Please press Ctrl+C once to exit vcclient.
*************** EP Error ***************
EP Error D:\a\_work\1\s\onnxruntime\core\providers\dml\dml_provider_factory.cc(491)\onnxruntime_pybind11_state.pyd!00007FFDA387DF4D: (caller: 00007FFDA387DD00) Exception(1) tid(296c) C0262002 Specified display adapter handle is invalid.
 when using ['DmlExecutionProvider']
Falling back to ['CPUExecutionProvider'] and retrying.
****************************************
*************** EP Error ***************
EP Error D:\a\_work\1\s\onnxruntime\core\providers\dml\dml_provider_factory.cc(491)\onnxruntime_pybind11_state.pyd!00007FFDA387DF4D: (caller: 00007FFDA387DD00) Exception(2) tid(296c) C0262002 Specified display adapter handle is invalid.
 when using ['DmlExecutionProvider']
Falling back to ['CPUExecutionProvider'] and retrying.
****************************************
[Voice Changer] pytorch failed to detect device. fallback to cpu
Torch not compiled with CUDA enabled
torch\nn\utils\weight_norm.py:134: FutureWarning: `torch.nn.utils.weight_norm` is deprecated in favor of `torch.nn.utils.parametrizations.weight_norm`.
  WeightNorm.apply(module, name, dim)
*************** EP Error ***************
EP Error D:\a\_work\1\s\onnxruntime\core\providers\dml\dml_provider_factory.cc(491)\onnxruntime_pybind11_state.pyd!00007FFDA387DF4D: (caller: 00007FFDA387DD00) Exception(3) tid(296c) C0262002 Specified display adapter handle is invalid.
 when using ['DmlExecutionProvider']
Falling back to ['CPUExecutionProvider'] and retrying.
****************************************
`

<img width="1331" height="868" alt="Image" src="https://github.com/user-attachments/assets/5c5e701f-5e4f-4d4c-84f4-b0daebd3ad71" />

**Comment by Devion0Zero** [comment]
How did you fix it??

**Comment by Haorelian** [comment]
> How did you fix it??

I couldn't still it doesn't use my GPU even though I can select it the usage doesn't ramp up and my CPU usage goes to %80.

**Comment by solitario001** [comment]
me too linux mint zena rx 9060 xt ...
